ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -312,13 +316,22 @@ def _full_backup_restore_with_DNS_zone(self, reinstall=False):
                                      '--uninstall',
                                      '-U'])
 
+            self.master.run_command([pkg_installer, '-y',
+                                     'remove', '*ipa-server-dns'])
```

Comment:
The method tasks.uninstall(self.master, ['*ipa-server-dns']) can be used instead.